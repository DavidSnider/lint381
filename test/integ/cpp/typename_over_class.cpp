template <class T>
class List {};

template <typename T>
class Array {};

template << foo
