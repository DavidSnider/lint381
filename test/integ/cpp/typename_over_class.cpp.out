typename_over_class.cpp:1:11: error: Use 'typename' instead of 'class' for template parameters
template <class T>
          ^^^^^
